### DISCUSSION TITLE
Integrazione Fatture in cloud con software gestionale

### DISCUSSION THREAD

**Discussion by SalvoDC** [fattureincloud/api#326] [Q&A] [answered]
Salve,
nel nostro gestionale gestiamo l' invio delle fatture elettroniche utilizzando oltre ad un nostro servizio anche quelli di terze parti per venire incontro alle necessità dei clienti. Ultimamente si è posta la necessità di integrare, ove possibile anche i servizi di Fatture in CLoud.
Nello specifico serve avere la possibilità di poter inviare fatture in formato  XML creato dal nostro SW tramite uno dei metodi delle API2.0 e poter scaricare la lista dei documenti ricevuti e dei relativi importi.

Ho già controlalto diverse volte i metodi delle API 2.0 ma non ho trovato alcun metodo che passato un XML già creato invii quest'ultimo. Non lo trovo io o non c'è? E in questo ultimo caso come bisogna operare per inviare un documento(Fattura o altro ) tramite le API2.0? 
Per quanto riguarda la lista dei documenti emessi credo sia necessario il metodo GET /c/{company_id}/issued_documents, confermate? Grazie millee buon lavoro a tutti

**Comment by valmoz** [comment, marked as answer]
Buongiorno,
non c'è modo per inviare un XML creato da software esterni. Può però procedere a creare nuove fatture con il metodo [Create Issued Document](https://developers.fattureincloud.it/api-reference/#post-/c/-company_id-/issued_documents).
Questa guida potrebbe esserle utile: [Invoice Creation](https://developers.fattureincloud.it/docs/guides/invoice-creation)

Per la lista dei documenti emessi, può utilizzare il metodo [List Issued Documents](https://developers.fattureincloud.it/api-reference/#get-/c/-company_id-/issued_documents)

Grazie
Mauro
